src/languages/ruby.ts

```diff
@@ -1,15 +1,24 @@
 import {
+  ancestorChainNodeMatcher,
   argumentMatcher,
+  cascadingMatcher,
   conditionMatcher,
   createPatternMatchers,
+  matcher,
+  patternMatcher,
   trailingMatcher,
 } from "../util/nodeMatchers";
-import { NodeMatcherAlternative, ScopeType, SelectionWithEditor } from "../typings/Types";
+import {
+  NodeMatcherAlternative,
+  ScopeType,
+  SelectionWithEditor,
+} from "../typings/Types";
 import { SyntaxNode } from "web-tree-sitter";
 import { getNodeRange } from "../util/nodeSelectors";
+import { ancestorChainNodeFinder, patternFinder } from "../util/nodeFinders";
```

Comment:
heyo - `ancestorChainNodeFinder` isn't referenced. I'll fix it in my PR.  Looking more at this, still trying to parse it. Will merge to move the Ruby PR forward.

Reply:
oh right yeah sorry I replaced it with `ancestorChainNodeMatcher` but then forgot to remove this one